WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -349,9 +403,21 @@ public void onModerateCommentForNote(Note note, CommentStatus newStatus) {
         finish();
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(final NotificationEvents.NotificationsRefreshCompleted event) {
+        setProgressVisible(false);
+        updateUIAndNote(false);
+    }
+
+    @SuppressWarnings("unused")
+    public void onEventMainThread(NotificationEvents.NotificationsRefreshError error) {
+        setProgressVisible(false);
+        ToastUtils.showToast(this, getString(R.string.error_refresh_notification));
```

Comment:
Should we close the activity if an error occurs retrieving notes, and the current note is null?